config/environment.js

```diff
@@ -47,6 +47,7 @@ module.exports = function (environment) {
       bestpracticessecurity: 'https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs',
       blog: 'https://blog.travis-ci.com',
       buildMatrix: 'https://docs.travis-ci.com/user/build-matrix/',
+      buildConfigValidation: 'https://docs.travis-ci.com/user/build-config-validation/',
```

Comment:
This link does not open for me, it shows Not Found page - is it expected? I assume that the documentation page might not be there yet, but I just wanted to bring it to your attention.

Reply:
yes, this is on purpose. it's going to be there once https://github.com/travis-ci/docs-travis-ci-com/pull/2548 has been merged